packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -752,9 +752,12 @@ export const audiusBackend = ({
   function getEthWeb3Config() {
     // In a dev env, always ignore the remote var which is inherited from staging
     const isDevelopment = env.ENVIRONMENT === 'development'
-    const providerUrls = isDevelopment
-      ? ethProviderUrls
-      : getRemoteVar(StringKeys.ETH_PROVIDER_URLS) || ethProviderUrls
+    const overrideRemoteVar =
+      ethProviderUrls !== undefined && ethProviderUrls.length > 0
+    const providerUrls =
+      isDevelopment || overrideRemoteVar
+        ? ethProviderUrls
+        : getRemoteVar(StringKeys.ETH_PROVIDER_URLS) || ethProviderUrls
```

Comment:
@raymondjacobson adding you for this change specifically since you added the env.stage.ts change. The remote urls in that file were previously being overridden by optimizely even locally. Is this expected behavior? Do we want to be favoring optimizely over locally set config?

Reply:
This change prioritizes what is in env.stage.ts by the way.

Reply:
Yes optimizely should take precedence over locally sent env!

Reply:
Should that configuration option be removed from env.stage.ts then? It's never used and could be misleading should it be set to something.

Reply:
okay I will remove this change then